conf/spgwu.bess

```diff
@@ -50,6 +50,8 @@ def scan_dpdk_ports():
             else:
                 raise
         idx += 1
+        if idx == 30:
```

Comment:
Any reason why the `idx` check is hard-coded till 30?

Reply:
@hyunsun somehow the accepted suggestion got disappeared.
@ajamshed 
```
 # RTE_MAX_ETHPORTS is 32 and we need 2 for vdevs
```

Reply:
Hmm.....I'll include it to my patch.